docs/linting/Typed_Linting.mdx

```diff
@@ -47,7 +47,7 @@ You may see new rules reporting errors based on type information!
 The `parserOptions.project` option can be turned on with either:
 
 - `true`: to always use `tsconfig.json`s nearest to source files
-- `string | string[]`: any number of glob paths to match TSConfig files relative to the
+- `string | string[]`: any number of glob paths to match TSConfig files relative to `parserOptions.tsconfigRootDir`, or the CWD if that is not provided
```

Comment:
> CWD

[Suggestion] My only concern here is that some users won't know what _"CWD"_ refers to. What do you think about explicitly saying it?

```suggestion
- `string | string[]`: any number of glob paths to match TSConfig files relative to `parserOptions.tsconfigRootDir`, or the current working directory if that is not provided
```

Reply:
Thanks for the review! Yes, sure that sounds good to me, please see [7cb4d25](https://github.com/typescript-eslint/typescript-eslint/pull/7052/commits/7cb4d2590ef3de758834c34b4e96bade01ba6234) 🙂